pkg/agent/flowexporter/exporter/exporter.go

```diff
@@ -551,3 +559,58 @@ func (exp *flowExporter) sendDataSet() (int, error) {
 	klog.V(4).Infof("Data set sent successfully. Bytes sent: %d", sentBytes)
 	return sentBytes, nil
 }
+
+func fetchPodSubnets(k8sClient kubernetes.Interface) ([]*net.IPNet, error) {
+	podSubnets := []*net.IPNet{}
+	nodeList, err := k8sClient.CoreV1().Nodes().List(context.TODO(), metav1.ListOptions{})
```

Comment:
As this is done as part of the initialization of Flow Exporter, can we be sure that all the nodes with Antrea agents are up?

Reply:
We discussed offline and current solution is if source/destination ip is not known we will do the query and update the podCIDRs.

Reply:
please integrate with the NodeRouteController, it already has this information

Reply:
Updated the PR integrating with the NodeRouteController.